dev/com.ibm.ws.microprofile.rest.client_fat_tck/publish/tckRunner/tck/src/test/resources/arquillian.xml

```diff
@@ -26,7 +26,9 @@
             <property name="verifyAppDeployTimeout">60</property>
             <property name="failSafeUndeployment">${tck_failSafeUndeployment}</property>
             <property name="appUndeployTimeout">${tck_appUndeployTimeout}</property>
-            <property name="javaVmArguments">-Dcom.ibm.tools.attach.enable=yes</property> <!-- this goes here because this tck project asks arquillian to start the server -->
+            <!-- com.ibm.tools.attach.enable=yes is for using the localConnector-1.0 feature on zOS -->
+            <!-- jdk.attach.allowAttachSelf is to allow late self-attach on JDK 11+ -->
+            <property name="javaVmArguments">-Dcom.ibm.tools.attach.enable=yes -Djdk.attach.allowAttachSelf=true</property>
```

Comment:
FYI: Don't think these should be needed since the arquillian container should be using the `server` script to start and stop the server since 1.0.5